docs/source/concepts/ElementalAnalysisUsingNegativeMuons.rst

```diff
@@ -0,0 +1,50 @@
+.. _Elemental analysis using negative muons:
+
+Elemental analysis using negative muons
+=======================================
+
+
+.. contents::
+  :local:
+  
+Overview
+--------
+This is a non-destructive , depth selective method of determining the elemental composition of a sample using characteristic Xrays 
+from  Muonic atoms.
+ 
+What happens when a muon is implanted in a sample?
+--------------------------------------------------
+* Muon is captured forming a mounic atom.
+* Muon cascades down atom to N = 1  (1s) energy level within ~10e-13 seconds emitting muonic x-rays [#HIL1]_.
```

Comment:
you should use maths formatting: https://github.com/mantidproject/mantid/blob/master/docs/source/algorithms/EstimateMuonAsymmetryFromCounts-v1.rst